shared/chat/inbox/row/index.tsx

```diff
@@ -42,7 +43,7 @@ const makeRow = (
             conversationIDKey={item.conversationIDKey}
             layoutIsTeam={item.isTeam}
             layoutName={item.teamname}
-            isSelected={item.selected}
```

Comment:
this isnt' part of the items now since that causes them to all thrash when this changes